packages/client/src/helpers/client/interceptors/authentication/token-providers/__tests__/TokenData.test.js

```diff
@@ -1,7 +0,0 @@
-import TokenData from '../TokenData';
```

Comment:
Question: Why was this test removed?

Reply:
While in JS we needed to alert the developer whenever he declares a new TokenData without the data. While migrating to TS, the data prop is required in the TokenData.ts file, so it prints an error type, having no need to throw this error thanks to the typescript capabilities.

Reply:
We recovered this test file.